docs/usage.md

```diff
@@ -38,12 +38,14 @@ The first step in creating hermetic builds is to fetch the dependencies for one
 cachi2 fetch-deps \
   --source ./foo \
   --output ./cachi2-output \
+  --sbom-output-type cyclonedx \
   '{"path": ".", "type": "<supported package manager>"}'
 ```
 
-* `--source` - the path to a *git repository* on the local disk
-* `--output` - the path to the directory where Cachi2 will write all output
-* `{JSON}`   - specifies a *package* (a directory) within the repository to process
+* `--source`           - the path to a *git repository* on the local disk `[default: .]`
+* `--output`           - the path to the directory where Cachi2 will write all output `[default: ./cachi2-output]`
+* `--sbom-output-type` - the format of generated SBOM, supports cyclonedx and spdx. `[default: cyclonedx]`
```

Comment:
A clarity nitpick: `spdx` and `cyclolnedx` are supported values, [CycloneDX](https://cyclonedx.org/)(of version 1.4) and [SPDX](https://spdx.github.io/spdx-spec/v2.3/)(of version 2.3) are supported formats. Could you please update this line accordingly to something like

```
`--sbom-output-type` - the format of generated SBOM, supported values are `cyclonedx` (outputs [CycloneDX v1.4](https://cyclonedx.org/docs/1.4/json)) and `spdx` (outputs [SPDX v2.3](https://spdx.github.io/spdx-spec/v2.3/)). `[default: cyclonedx]`
```

Another thing worth explicitly mentioning is where to find the SBOM. Something along these lines:

```
The resulting SBOM will be stored as a JSON in `bom.json` in output directory.
```